### PR TITLE
First task

### PR DESCRIPTION
**Pull request by 1lyan**
This is the first task in the nodejs course

### CODE REVIEW

File: config.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  APP_PORT: process.env.APP_PORT,
+  ENV: process.env.ENV,
```
Comment (rfist) [inline config.js]: env must be passed as a cli argument, so command `node app.js testenv` will give you output `env is testenv`
what are you looking for is `process.argv`

File: app.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+}
+
+const http = require('http');
+
+const message = `Listening on port: ${config.APP_PORT}. ENV is ${config.ENV}`;
+http.createServer((request, response) => {
+    console.log('New incoming request');
+    response.writeHeader(200, { 'Content-Type': 'application/json' });
+    response.end(JSON.stringify({ message: 'Hello, world!' }));
+}).listen(config.APP_PORT, () => console.log(message));
```
Comment (rfist) [inline app.js]: Please, create `logger.js` and import logging functions from it.

**Review by rfist — CHANGES_REQUESTED** [review]
Probably content-type `application/json` is too much for one short string, but we can live with it. 
Fix please other comments

**Review by rfist — APPROVED** [review]
Nice work!

**Comment by 1lyan** [comment]
> Nice work!

@rfist , thanks :)

**Pull request merged**